index.js

```diff
@@ -15,30 +20,35 @@ module.exports = plugin(function ({ addUtilities, theme, matchUtilities, addComp
         black: 'var(--color-text-black)',
         interactive: {
           default: 'var(--color-text-interactive-default)',
+          DEFAULT: 'var(--color-text-interactive-default)',
           hover: 'var(--color-text-interactive-hover)'
         }
       },
       backgroundColor: {
         base: {
           default: 'var(--color-background-base-default)',
+          DEFAULT: 'var(--color-background-base-default)',
```

Comment:
I was trying to use `bg-base` as documented in https://astro-tailwind-config.netlify.app/#Colors and noticed that it wasn't working. Try it out yourself, of course - but the case sensitivity here seems to be the issue.

I'm sure some people just used `bg-base-default` and moved on with their lives, but where's the fun in that.

That being said, we should maintain backward compatibility for those folks and retain the `bg-base-default` so we don't break them (until a major version bump).